src/Psalm/Internal/Cli/Psalm.php

```diff
@@ -212,7 +212,7 @@ public static function run(array $argv): void
         $include_collector = new IncludeCollector();
         $first_autoloader = $include_collector->runAndCollect(
             // phpcs:ignore SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName
-            function () use ($current_dir, $options, $vendor_dir): ?ClassLoader {
```

Comment:
There were several return types like this in other entrypoint classes. Also, you may want to add a comment explaining why they should be left as they are.

Reply:
Thanks, fixed and documented